include/convex_bodies/ellipsoids.h

```diff
@@ -0,0 +1,170 @@
+#ifndef ELLIPSOIDS_H
+#define ELLIPSOIDS_H
+
+//#define CGAL_QP_NO_ASSERTIONS
+
+//this is for LP-solver
+#include <iostream>
+//#include <CGAL/basic.h>
+//#include <CGAL/QP_models.h>
+//#include <CGAL/QP_functions.h>
+// choose exact integral type
+//#ifdef CGAL_USE_GMP
+//#include <CGAL/Gmpzf.h>
+//typedef CGAL::Gmpzf ET;
+//#endif
+//typedef double ET;
+//#else
+//#include <CGAL/MP_Float.h>
+//typedef CGAL::MP_Float ET;
+//#endif
+//#include <boost/random/shuffle_order.hpp>
+//#include "../rref.h"
+//#include "LPsolve/solve_lp.h"
+//EXPERIMENTAL
+//to implement boundary oracles using NN queries  
+//#include <flann/flann.hpp>
+
+
+// ellipsoid class
+template <typename K>
+class ellipsoid{
```

Comment:
be consistent with naming, in the other files you used capital letter, so do the same here, i.e. ```Ellipsoid```